src/renderer/store/playback/reducer.ts

```diff
@@ -21,6 +24,7 @@ const playbackReducer: Reducer<ApplicationStore['playback'], Action<any>> = (
       isPlaying: playState.isPlaying,
       time: playback.time,
       lastUpdated: playState.lastUpdated,
+      rangeOverride: playback.rangeOverride,
```

Comment:
Can just use object destructuring (spreading) of previous object state rather than setting individual properties based on previous state.
eg:
![image](https://user-images.githubusercontent.com/69132311/192128080-9f843cd9-d98d-4208-a138-80409047b807.png)

Reply:
Yeah in this case something like

```
const { isPlaying, lastUpdated } = playState;

return {
  ...playback,
  isPlaying,
  lastUpdated,
}
```

would work. I think still be explicit about the things being updated rather than just going `{ ...playback, ...playState }`

Reply:
thanks Chloe for tagging along, I've tidied the reducers up and all of them now follow the same style you suggested. Here is the commit: [491b0c3](https://github.com/chloebrett/mlvet/pull/308/commits/491b0c33fb0b86a7d69062567a28691f293f0b1e)